test/test-77-compare-dicts-and-tests/main.js

```diff
@@ -6,18 +6,9 @@ const fs = require('fs');
 const path = require('path');
 const assert = require('assert');
 
-let configs = fs.readdirSync('../../dictionary');
-configs = configs
-  .filter(function (config) {
-    return config !== '.eslintrc.json';
-  })
-  .map(function (config) {
-    assert.strictEqual(config.slice(-3), '.js');
-    return config.slice(0, -3);
-  });
-
-let tests = fs.readdirSync('../test-79-npm');
-tests = tests.filter(function (test) {
+const configs = Object.keys(require('../../lib-es5/dictionary').default);
+
+const tests = fs.readdirSync('../test-79-npm').filter(function (test) {
```

Comment:
There's not really an advantage here to inlining the readdir and the filter, I'd rather you extract the filter function into a named function defining what the filter is actually doing here, like `excludeIsolatorTests` or something

Reply:
I don't see any benefit from doing that and I don't think we should touch anything more than what this PR does, especially with old tests like this.

Reply:
I meant if you are going to change these lines anyway, why not answer some of the questions posed by the obscure `.filter` logic...what is that filtering doing? simply breaking that filter function into a named function that actually tells the reader what it's doing, or why would help the next person come in and also continue to improve the code next time they touch this line of code.

Reply:
I see, I've pushed some changes for this file, did I understand you correctly with those changes?